keps/sig-auth/1513-certificate-signing-request/README.md

```diff
@@ -272,6 +273,27 @@ Cluster admins can either:
   to ensure version-independent authorization policy grants consistent permissions
   across v1beta1 and v1 versions)
 
+#### Importance of .spec.signerName
+
+The CSR v1 API uses the `.spec.signerName` to clarify the usage of all well-known signers provided by Kubernetes.
+Each signer has a specific purpose and limitations in our [public docs](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers).
+The most noteworthy clarifications are around certificates signed for purposes other than client certificates valid
+against the `kube-apiserver`, `kubelet` certificates, and trust distribution.
+
+1. Client certificates were signed and became valid kube-apiserver credentials.
+   If a different actor wanted to terminate client credentials for itself, using the CSR API was an abuse that created
+   additional client certificates valid against the `kube-apiserver` which probably wasn't intended.
```

Comment:
abuse might be a bit strong... calling out the unintentional validity against kube-apiserver is probably sufficient:

```suggestion
   If a different actor wanted to terminate client credentials for itself, using the CSR API created
   additional client certificates valid against the `kube-apiserver` which probably wasn't intended.
```

Reply:
updated